### PR TITLE
Feature tests Trades and Trades Details

### PR DESCRIPTION
**Pull request by MoyasiGinko**
In this milestone,

- Created a new branch `feature-tests-trades`
- Implemented test for TradesLists
- Implemented test for TradesDetails
- All tests are passing ✅

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "@fortawesome/fontawesome-svg-core": "^6.4.2",
     "@fortawesome/free-brands-svg-icons": "^6.4.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
-    "@reduxjs/toolkit": "^1.9.6",
+    "@reduxjs/toolkit": "^1.9.7",
```
Comment (lincoln1883) [inline package.json:9]: Please put the redux/toolkit on line 13.

File: src/__test__/App.test.js
Comment (lincoln1883) [inline src/__test__/App.test.js:1]: You can also delete this file it's making the test fail.

Screenshot from 2023-10-09 12-32-03

File: src/__test__/TradesDetails.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      tradeDetails: {
+        trade: null,
+      },
+    });
+
+    render(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={['/trade/1']}>
+          <Routes>
+            <Route path="/trade/:id" element={<TradesDetails />} />
```
Comment (lincoln1883) [inline src/__test__/TradesDetails.test.js:70]: I got an error in this line.

Screenshot from 2023-10-09 12-31-39

File: src/components/ReserveTdForm.jsx [hunk 3]
```diff
@@ -42,6 +42,7 @@ const ReserveTdForm = () => {
       .unwrap()
       .then(() => {
         // Handle success here if needed
+        // console.log('Reservation created!');
```
Comment (lincoln1883) [inline src/components/ReserveTdForm.jsx:45]: ?

**Review by lincoln1883 — CHANGES_REQUESTED** [review]

File: src/__test__/TradesDetails.test.js (hunk 2 above)
Comment (MoyasiGinko) [reply src/__test__/TradesDetails.test.js:70]: please install the dependencies first by `npm install` before running the tests, its working in my environment

**Review by lincoln1883 — APPROVED** [review]
@MoyasiGinko ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All tests are passing ✅
- No linting errors 👏
- Made change recommended 👍🏿  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**